samples/Common.c

```diff
@@ -361,14 +361,31 @@ STATUS initializePeerConnection(PSampleConfiguration pSampleConfiguration, PRtcP
             }
         }
     }
-
+    pSampleConfiguration->iceUriCount = uriCount + 1;
     CHK_STATUS(createPeerConnection(&configuration, ppRtcPeerConnection));
 
 CleanUp:
 
     return retStatus;
 }
 
+// Return ICE server stats for a specific streaming session
+STATUS getIceServerStats(PSampleStreamingSession pSampleStreamingSession)
+{
+    ENTERS();
+    STATUS retStatus = STATUS_SUCCESS;
+    RtcStats rtcmetrics;
+    UINT32 j = 0;
+    rtcmetrics.requestedTypeOfStats = RTC_STATS_TYPE_ICE_SERVER;
+    for (; j < pSampleStreamingSession->pSampleConfiguration->iceUriCount; j++) {
```

Comment:
I might've missed something, but how are the metrics returned? I saw that rtcmetrics is defined locally.

Reply:
The metrics get logged as and when the call is made. The rtcmetrics is populated when call to `rtcPeerConnectionGetMetrics` is made.

Reply:
right, but rtcmetrics is not being used anywhere. As soon as the function ends, rtcmetrics will be removed from the stack.

Reply:
Yes. In this case, we are not using it anywhere. It could be used right after the call. 
Now that I think about it, I probably can log in the application itself and not depend on internal logging. Let me do that.

Reply:
I like this approach better! Less reliant on environment variables, makes it much more explicit to the user.